masq_lib/src/constants.rs

```diff
@@ -1,7 +1,10 @@
 // Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
 
 use crate::blockchains::chains::Chain;
+use crate::combined_parameters::{PaymentCurves, RatePack, ScanIntervals};
 use const_format::concatcp;
+use lazy_static::lazy_static;
+use std::time::Duration;
 
 pub const DEFAULT_CHAIN: Chain = Chain::EthMainnet;
```

Comment:
Do we want EthMainnet as default or PolygonMainnet?

Reply:
> Do we want EthMainnet as default or PolygonMainnet?

I would suggest we default to polygon mainnet @bertllll